cmd/nerdctl/restart_linux_test.go

```diff
@@ -69,3 +70,28 @@ func TestRestartPIDContainer(t *testing.T) {
 
 	assert.Equal(t, baseOutput, sharedOutput)
 }
+
+func TestRestartWithTime(t *testing.T) {
+	t.Parallel()
+	base := testutil.NewBase(t)
+	tID := testutil.Identifier(t)
+
+	base.Cmd("run", "-d", "--name", tID, testutil.AlpineImage, "sleep", "infinity").AssertOK()
+	defer base.Cmd("rm", "-f", tID).AssertOK()
+	base.EnsureContainerStarted(tID)
```

Comment:
Sorry, but same here this is useless too

Reply:
Thanks :)